docs/plugins/start_using_plugins/finding_and_installing_plugins.md

```diff
@@ -10,12 +10,6 @@ metadata, to produce a more user friendly way to find napari plugins.
 Similarly, plugins annotated on PyPI with `Framework :: napari` are listed in
 the `Plugins > Install/Uninstall Plugins` menu within napari.
 
-## Finding plugins on the napari hub
-
-The [napari hub](https://napari-hub.org) hosts information about all plugins.
-You can browse, search, and filter to find plugins that fit your needs.
-You can also share links to specific search results and plugins.
-
```

Comment:
Caught my eye and I can't suggest on the right line, but below in `Installing plugins` new line 22:
Could you mention:
- scrolling through the list of plugins or filtering plugins using the text entry box at the top
- just clicking `install` in the tile of a plugin
This to not just focus on the tricker manual install as is currently.